src/pages/SearchPage/index.js

```diff
@@ -93,7 +94,7 @@ function SearchPage({betas, reports, isSearchingForReports}) {
 
         if (recentReports.length > 0) {
             newSections.push({
-                data: recentReports,
+                data: _.map(recentReports, (report) => ({...report, isBold: report.isUnread})),
```

Comment:
This caused inconsistency with LHN where the bold style was applied based on other factors too (the notification preference of the report)

https://github.com/Expensify/App/blob/2093642cebda7ef2ecf1f03958b2390463557f6b/src/components/LHNOptionsList/OptionRowLHN.tsx#L67